src/component/Nav/Modal/Modal.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
+import styled from 'styled-components';
+import { css } from 'styled-components';
+
+const Modal = ({ onOffModal, name, children, id }) => {
+  const stop = e => {
+    e.stopPropagation();
+  };
+
+  const handlemodal = ({ target }) => {
+    onOffModal(target.id);
+  };
+  return (
+    <ModalCantainer name={name} id={id} onClick={handlemodal}>
```

Comment:
```suggestion
    <ModalContainer name={name} onClick={handleModal}>
```
id라는 props는 사용되지 않는 것 같고, 변수명에 오타가 있습니다.

Reply:
다른 컴포넌트에서 사용해야해서 넣었습니다 ! 공용컴포넌트로 옮기겠습니다 !!